crates/holochain/src/core/workflow/sys_validation_workflow/tests.rs

```diff
@@ -37,31 +37,28 @@ async fn sys_validation_workflow_test() {
 
 #[cfg(feature = "unstable-warrants")]
 #[tokio::test(flavor = "multi_thread")]
-async fn sys_validation_produces_invalid_chain_warrant() {
+async fn sys_validation_produces_invalid_chain_op_warrant() {
+    use crate::test_utils::retry_fn_until_timeout;
+
     holochain_trace::test_run();
     let zome = SweetInlineZomes::new(vec![], 0);
     let (dna, _, _) = SweetDnaFile::unique_from_inline_zomes(zome).await;
 
-    let mut conductors = SweetConductorBatch::from_standard_config(2).await;
-    let ((alice,), (bob,)) = conductors
```

Comment:
No need having two conductors, because all we want is to fixturate an invalid op.